src/test/javascript/spec/component/shared/clone-repo-button.component.spec.ts

```diff
@@ -337,6 +346,49 @@ describe('CloneRepoButtonComponent', () => {
         expect(component.activeParticipation?.id).toBe(expected);
     });
 
+    it('should retrieve user from the server when VCS access token is missing', fakeAsync(() => {
+        component.user.vcsAccessToken = null;
```

Comment:
```suggestion
        component.user.vcsAccessToken = undefined;
```
Multiple times, causes client test failure.

Reply:
Interesting, I'll have a look.